templates/emails/customer-invoice.php

```diff
@@ -21,17 +21,37 @@
 }
 
 /**
+ * Executes the e-mail header.
+ *
  * @hooked WC_Emails::email_header() Output the email header
  */
 do_action( 'woocommerce_email_header', $email_heading, $email ); ?>
 
 <?php if ( $order->has_status( 'pending' ) ) : ?>
-	<p><?php printf( __( 'An order has been created for you on %1$s. To pay for this order please use the following link: %2$s', 'woocommerce' ), get_bloginfo( 'name', 'display' ), '<a href="' . esc_url( $order->get_checkout_payment_url() ) . '">' . __( 'pay', 'woocommerce' ) . '</a>' ); ?></p>
+	<p>
+	<?php
+	printf(
+		wp_kses(
+			/* translators: %1s item is the name of the site, %2s is a html link */
+			 __( 'An order has been created for you on %1$s. To pay for this order please use the following link: %2$s', 'woocommerce' ),
+			esc_html( get_bloginfo( 'name', 'display' ) ),
+			'<a href="' . esc_url( $order->get_checkout_payment_url() ) . '">' . esc_html__( 'Submit payment online.', 'woocommerce' ) . '</a>'
+		),
+		array(
+			'a' => array(
+				'href' => array(),
+			),
+		)
```

Comment:
This is an interesting way of doing `printf`. I wasn't able to find any info about this format, but it doesn't seem to be valid. Please send me a link if it's a valid way of doing printf, but right now I get these errors in the email template:
```
Notice: Array to string conversion in /srv/www/wordpress-default/public_html/wp-content/plugins/woocommerce/templates/emails/customer-invoice.php on line 43

Warning: printf(): Too few arguments in /srv/www/wordpress-default/public_html/wp-content/plugins/woocommerce/templates/emails/customer-invoice.php on line 43
```

Reply:
I'll fix it up and should have tested it; I was trying to keep the unit tests happy since the code doesn't meet standards and errors out since it doesn't. A fix is incoming.